ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -1131,22 +1133,27 @@ const triggerSet: TriggerSet<Data> = {
           trailblaze: {
             en: 'Wait => ${dir}',
             de: 'Warte => ${dir}',
+            fr: 'Attendez -> ${dir}',
           },
           trailblazeKnockback: {
             en: '${dir} Knockback',
             de: '${dir} Rückstoß',
+            fr: '$(dir) Poussée',
```

Comment:
you have normal bracket instead of curly ones `$(dir)` vs `${dir}`

Reply:
Oh god, I'm blind ... end of the week ...
Thanks a lot Akurosia